packages/apis/uniswapv2/src/query/trade.ts

```diff
@@ -22,18 +22,26 @@ import {
   pairOutputNextPair,
 } from "./pair";
 import Fraction from "../utils/Fraction";
-import { tokenAmountEquals, tokenEquals } from "./token";
+import { currencyEquals, tokenAmountEquals, tokenEquals } from "./token";
 import { PriorityQueue } from "../utils/PriorityQueue";
 import { TradeOptions } from "../utils/TradeOptions";
 import { ETHER } from "../utils/Currency";
+import { wrapIfEther } from "../utils/utils";
 
 import { BigInt } from "as-bigint";
+import { BigFloat } from "as-bigfloat";
+import { Nullable } from "@web3api/wasm-as";
 
 export function createTrade(input: Input_createTrade): Trade {
   const amounts: TokenAmount[] = new Array(input.route.path.length);
-  const nextPairs: Pair[] = new Array(input.route.pairs.length);
+  const nextPairs: Pair[] = new Array(input.route.pairs.length); // TODO this array is created and updated, but never read from. Do we need it?
```

Comment:
Can you delete if it is not used?

Reply:
I deleted it. I didn't delete it before because I wasn't sure if there was a reason it was added.